keras_cv/models/weights.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+import tensorflow as tf
+from keras.utils import data_utils
+
+
+def parse_weights(weights, include_top, model_type):
+    if not weights or tf.io.gfile.exists(weights):
+        return weights
+    if weights in WEIGHTS_CONFIG[model_type]:
+        if not include_top:
+            weights = weights + "-notop"
+        return data_utils.get_file(
+            origin=f"{BASE_PATH}/{model_type}/{weights}.h5",
+            cache_subdir="models",
+            file_hash=WEIGHTS_CONFIG[model_type][weights],
+        )
+
+    raise ValueError(
+        "The `weights` argument should be either `None`, a the path to the "
+        "weights file to be loaded, or the name of pre-trained weights from "
+        "`weights.py`.  Invalid `weights` argument: {weights}"
```

Comment:
lets provide users with a link here.  For now, we can link to the literal github file, in the future we can link to a keras.io documentation page.

If we don't want to include this yet, you can leave TODO(ianjjohnson) in a comment to fix this with a keras.io link